src/contentmanager.cpp

```diff
@@ -70,7 +70,7 @@ ContentManager::ContentManager(Library* library, kiwix::Downloader* downloader,
     connect(treeView, &QTreeView::doubleClicked, this, &ContentManager::openBookWithIndex);
     connect(&m_remoteLibraryManager, &OpdsRequestManager::languagesReceived, this, &ContentManager::updateLanguages);
     connect(&m_remoteLibraryManager, &OpdsRequestManager::categoriesReceived, this, &ContentManager::updateCategories);
-    setLanguages();
+    setCategories();
```

Comment:
This comment is for the commit message, not this line.

Commit message say : `If local library is displayed, we get the categories from LibraryDumper`
It is false as the categories are taken from Library itself (and it is a good thing)

Reply:
Fixed in history